src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -237,7 +236,7 @@ export class SavedObjectsService
     }));
 
     const createSORepository = (callCluster: APICaller, extraTypes: string[] = []) => {
-      return createRepository(
+      return SavedObjectsRepository.createRepository(
```

Comment:
NIT: maybe `#create()`  is explicit enough as it now belongs to the associated class?

Reply:
Yes, but we have a conflict with the public method #create to create saved objects, so I had to be more creative with the name.